package.json

```diff
@@ -16,10 +16,11 @@
   },
   "scripts": {
     "test": "tree-sitter test",
-    "format": "prettier --trailing-comma es5 --write grammar.js",
-    "format-check": "prettier --trailing-comma es5 --check grammar.js"
+    "format": "prettier --trailing-comma es5 --write grammar.js && clang-format -i src/scanner.c",
+    "format-check": "prettier --trailing-comma es5 --check grammar.js && cat src/scanner.c | clang-format src/scanner.c | diff src/scanner.c -"
   },
   "dependencies": {
+    "clang-format": "^1.8.0",
```

Comment:
Let's install as dev dependency.

Reply:
done, also formatted the file per the format file I added